src/view/components/selected-devices/selected/useSelectedDevices.ts

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { useRootContext } from "../../../provider/useRootContext";
+
+export const useSelectedDevices = () => {
+    const {
+        selectedDevicesContext: {
+            selectedDevices,
+            removeSelectedDevice,
+            setDevicePercentage,
+        },
+    } = useRootContext();
+    const [averagePercentage, setWorldPercentage] = useState(1);
```

Comment:
fraction?

Reply:
I am intending to keep percentage in the name, so will need to refactor values to 100 - have added it to a separate ticket